codice/alliance
# CAL-413 Convert NitfPreStoragePlugin to NitfPostIngestPlugin

#### What does this PR do?
Merge AFTER https://github.com/codice/ddf/pull/3276 is merged.
- Cherry picks 85dde9e8 and 991c2567
- Small constant change

#### Who is reviewing it? 
@roelens8 @emmberk @kcover  

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@clockard
@coyotesqrl

#### How should this be tested?
Verify using CDM in place monitoring ingest NITFs. Verify ingested NITFs can have both derived products downloaded, the original product downloaded, and verify NITF chipping.

#### Any background context you want to provide?

#### What are the relevant tickets?
[CAL-413](https://codice.atlassian.net/browse/CAL-413)

#### Screenshots (if appropriate)
#### Checklist:
